arc/src/zjs_ashell_arc.json

```diff
@@ -1,8 +1,8 @@
 {
     "module": "ashell_arc",
-    "depends": ["i2c",
+    "depends": ["aio_arc",
+                "grove_lcd_arc",
                 "i2c_arc",
```

Comment:
i think you need the i2c_arc here in ashell_arc, as it's required since we depend on i2c in the x86 ashell.json,  so the i2c_arc is not a dependency of grove_lcd.